0000-template.md

```diff
@@ -1,74 +0,0 @@
----

```

Comment:
The original template should be kept for future RFCs.

Reply:
Committed template file to master.

Reply:
See the second bullet of https://git.soma.salesforce.com/aura/aura/blob/master/aura-spring/src/main/java/org/auraframework/system/Annotations.java#L74:
> Copy 0000-template.md to text/0000-my-feature.md (where 'my-feature' is descriptive. Don't assign an RFC number yet).

This needs to be fixed immediately because you'll lose comment history.